ops/op2/dispatch_slow.rs

```diff
@@ -377,6 +377,19 @@ pub fn from_arg(
         };
       }
     }
+    Arg::OptionString(Strings::RefStr) => {
```

Comment:
Is this on purpose?

Reply:
Whoops - nope. That's for a different PR!